bitnami/mysql/values.yaml

```diff
@@ -214,7 +214,7 @@ primary:
   ##
   configuration: |-
     [mysqld]
-    default_authentication_plugin={{- .Values.auth.defaultAuthenticationPlugin | default "mysql_native_password" }}
+    authentication_policy= '{{- .Values.auth.defaultAuthenticationPlugin | default "caching_sha2_password" }}'
```

Comment:
```suggestion
    authentication_policy={{- .Values.auth.defaultAuthenticationPlugin | default "caching_sha2_password" }}
```

Reply:
The single quotes are needed. I am removing the white space